packages/e2e/README.md

```diff
@@ -130,10 +130,11 @@ documentation](https://firebase.google.com/docs/test-lab/?gclid=EAIaIQobChMIs5qV
 to set up a project.
 
 To run an e2e test on Android devices using Firebase Test Lab, use gradle commands to build an
-instrumentation test for Android.
+instrumentation test for Android, after creating `androidTest` as suggested in the last section.
 
-```
+```bash
 pushd android
+flutter build apk
```

Comment:
I assume this is to set the desired runtime mode?

If so we should explain that. If not we should explain why we're doing this

Reply:
I don't think this set the runtime mode. This routine still uploads debug app (which I still need to figure out how to get a release/profile testing app), but `flutter build` generates some file and directory, e.g. `./gradlew`. I'm not sure if there's  anything else.